scrapy_ft_jobs_sites/scrapy_ft_jobs_sites/spiders/indeedin.py

```diff
@@ -37,10 +37,13 @@ def __init__(self, *args, **kwargs):
             self.start_urls.append(URL)
 
             for pagination in range(10, 60, 10):
-                URL = self.base_url_pattern + item.replace(" ", "+") + "&start=" + str(pagination)
+                URL = self.base_url_pattern + \
+                    item.replace(" ", "+") + "&start=" + str(pagination)
                 self.start_urls.append(URL)
 
     def parse_item(self, response):
```

Comment:
Make sure you have test case for parsing logic, this will help making sure our standard is enforced